deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +204,62 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payloadEventsString: __ENV.DATA_BTM_SQS_PAYLOAD_EVENTS,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadEventsArray = JSON.parse(eventData.payloadEventsString)
+
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
+let messageBody: string = ''
+let randomTimestamp: number
+let timestampFormatted: string
+
 const sqs = new SQSClient(awsConfig)
 
-export function sendEvent (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
-  sqs.sendMessage(env.sqs_queue, messageBody)
+export function sendEventType1 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[0])
+  sendSQSMessage()
+}
+
+export function sendEventType2 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[1])
+  sendSQSMessage()
+}
+
+export function sendEventType3 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[2])
+  sendSQSMessage()
+}
+
+export function sendEventType4 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[3])
+  sendSQSMessage()
+}
+
+export function sendEventType5 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[4])
+  sendSQSMessage()
+}
+
+export function sendEventType6 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[5])
+  sendSQSMessage()
+}
+
+export function sendEventType7 (): void {
+  messageBody = JSON.stringify(payloadEventsArray[6])
+  sendSQSMessage()
 }
 
-export function sendEventDebug (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
+export function sendSQSMessage (): void {
```

Comment:
```suggestion
export function sendSQSMessage (messageBody: string): void {
```
Pass the message bodies in as a parameter

Reply:
Thanks Done